### PR TITLE
Submitting technote draft: 2018-07-10-phylogram

### PR DESCRIPTION
**Pull request by shaunpwilkinson**
Hi @sckott and @stefaniebutland, here is the first draft for the phylogram technote. For some reason I couldn't get it to build with a future date. Also do you have a topic id you'd like me to put in the YAML header? Cheers S

### CODE REVIEW

**Comment by shaunpwilkinson** [comment]
Scratch that, the future date seems to deploy fine with Netlify, just wouldn't work locally with hugo.

File: content/technotes/2018-07-17-phylogram.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+---
+slug: "phylogram"
+title: "phylogram: dendrograms for evolutionary analysis"
+package_version: 2.1
+authors:
+  - name: Shaun Wilkinson
+    url: https://github.com/shaunpwilkinson
+date: 2018-07-17
+categories: technote
+topicid: 1175
```
Comment (stefaniebutland) [inline content/technotes/2018-07-17-phylogram.md]: omit the `1175` from topicid. I'll add that right before publication to enable commenting

File: content/technotes/2018-07-17-phylogram.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+dendextend::tanglegram(dndlist, fast = TRUE, margin_inner = 5)
+```
+
+<img src="/img/blog-images/2018-07-17-phylogram/unnamed-chunk-8-1.png" width="1000px" style="display: block; margin: auto auto auto 0;" />
+
+\
+
+Hopefully users will find the package useful for a range of other applications, 
+and I look forward to receiving feedback from the community. 
+Thanks to [wcornwell](https://github.com/wcornwell) and
```
Comment (stefaniebutland) [inline content/technotes/2018-07-17-phylogram.md]: Please write their names in full as Will Cornwell, Ben J. Ward, Scott Chamberlain

File: content/technotes/2018-07-17-phylogram.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+dndlist <- dendextend::dendlist(dnd1, dnd3)
+dendextend::tanglegram(dndlist, fast = TRUE, margin_inner = 5)
+```
+
+<img src="/img/blog-images/2018-07-17-phylogram/unnamed-chunk-8-1.png" width="1000px" style="display: block; margin: auto auto auto 0;" />
+
+\
+
+Hopefully users will find the package useful for a range of other applications, 
+and I look forward to receiving feedback from the community.
```
Comment (stefaniebutland) [inline content/technotes/2018-07-17-phylogram.md]: Be specific on how you would like feedback and what kind of input is helpful. Maybe suggest people open an [issue](https://github.com/ropensci/phylogram/issues) if that's what you'd like

File: content/technotes/2018-07-17-phylogram.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+
+
+Evolutionary biologists are increasingly using R for building, 
+editing and visualizing phylogenetic trees. 
+The reproducible code-based workflow and comprehensive array of tools 
+available in packages such as [ape](http://ape-package.ird.fr/), 
+[phangorn](https://github.com/KlausVigo/phangorn) and 
+[phytools](http://blog.phytools.org/) make R an ideal platform for
+phylogenetic analysis. 
+Yet the many of the different tree formats are not well integrated,
```
Comment (stefaniebutland) [inline content/technotes/2018-07-17-phylogram.md]: s/Yet the many of/Yet many of/

File: content/technotes/2018-07-17-phylogram.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+```r
+x <- list(1, list(2, 3))
+## set class, midpoint, members and leaf attributes for each node
+x <- remidpoint(x)
+## set height attributes for each node
+x <- as.cladogram(x)
+```
+
+
+A nice feature of the dendrogram object type that tree
```
Comment (stefaniebutland) [inline content/technotes/2018-07-17-phylogram.md]: s/object type that/object type is that/

File: content/technotes/2018-07-17-phylogram.md [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+  return(node)
+}
+x <- dendrapply(x, set_label)
+plot(x, horiz = TRUE)
+```
+
+<img src="/img/blog-images/2018-07-17-phylogram/unnamed-chunk-6-1.png" width="1000px" style="display: block; margin: auto auto auto 0;" />
+
+
+### Applications
```
Comment (stefaniebutland) [inline content/technotes/2018-07-17-phylogram.md]: I like the way you laid out this example

**Comment by stefaniebutland** [comment]
> Scratch that, the future date seems to deploy fine with Netlify, just wouldn't work locally with hugo.

I think @sckott said you need to `hugo serve -F` for local

**Comment by shaunpwilkinson** [comment]
Thanks @stefaniebutland ! That shouldn't be a problem to have it ready by then. Once I have @sckott 's edits I'll get them incorporated asap.

**Comment by sckott** [comment]
having a look now

**Comment by sckott** [comment]
Read through it, all looks good to me  ✔️

**Comment by stefaniebutland** [comment]
one last change request: can you please put "phylogram" package name in back ticks each time, (`phylogram`) and not boldface?

Look for a tweet from rOpenSci July 12 :-). Thank you again!

**Comment by shaunpwilkinson** [comment]
Thanks both! Have done the back-ticks now so should be good to go.

**Pull request merged**